### ISSUE TITLE
About  “use epoch based learning rate scheduler”

### ISSUE THREAD

**Issue by LinpengPan** [chaofengc/IQA-PyTorch#38]
Whether this method supports  “CosineAnnealingRestartLR”
